src/openms/source/FORMAT/HANDLERS/MzIdentMLDOMHandler.cpp

```diff
@@ -539,7 +539,8 @@ namespace OpenMS
         String unitCvRef = XMLString::transcode(param->getAttribute(XMLString::transcode("unitCvRef")));
         String type = XMLString::transcode(param->getAttribute(XMLString::transcode("type")));
         DataValue dv;
-        dv.setUnit(unitAcc + ":" + unitName);
+        // dv.setUnit(unitAcc + ":" + unitName);
+        // TODO
```

Comment:
I dont fully understand the DOM code and I am not sure whether this will be a problem or not.